observability/logging/scheduling_resources/logging-taints-tolerations.adoc

```diff
@@ -25,6 +25,3 @@ include::modules/cluster-logging-collector-pod-location.adoc[leveloffset=+1]
 ifdef::openshift-enterprise,openshift-origin[]
 * xref:../../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations[Controlling pod placement using node taints]
 endif::[]
-ifdef::openshift-rosa,openshift-dedicated[]
```

Comment:
Should this be ifndef rather than just removing it because it looks like it is in enterprise too just checking?